src/QueryFactory.php

```diff
@@ -34,6 +34,18 @@ public function create(): Query
                 )
             );
             $query->setFilter($filter);
+        } elseif ($this->config->hasFilter()) {
+            $query->setFilter(Filter::applyQueryString($this->config->getFilter()));
+        }
+
+        // Select
+        if ($this->config->hasSelect()) {
+            $query->setSelectFields($this->config->getSelect());
+        }
+
+        // Limit
+        if ($this->config->hasLimit()) {
+            $query->setTop($this->config->getLimit());
```

Comment:
limit/top -> tu sa to nastavuje do `$query` a znamena to limit vramci jedneho request/page, ... vid koment v `Extractor`.